doc/articles/controls/WebView.md

```diff
@@ -83,4 +83,44 @@ webView.WebMessageReceived += (s, e) =>
 };
 ```
 
-The `WebMessageAsJson` property contains a JSON-encoded string of the data passed to `postWebViewMessage` above.
\ No newline at end of file
+The `WebMessageAsJson` property contains a JSON-encoded string of the data passed to `postWebViewMessage` above.
+
+## Navigating to web content in application package
+
+To load local web content bundled with the application, you can use the `SetVirtualHostNameToFolderMapping` method. This allows you to set a virtual hostname that maps to a folder within the package, from which the web content will be loaded:
+
+```csharp
+await webView.EnsureCoreWebView2Async();
+webView.CoreWebView2.SetVirtualHostNameToFolderMapping(
+	"UnoNativeAssets",
+	"WebContent",
+	CoreWebView2HostResourceAccessKind.Allow);
+webView.CoreWebView2.Navigate("http://UnoNativeAssets/index.html");
+```
+
+This will navigate to the `index.html` file stored in the `WebContent` folder. This folder must be included in platform-specific location on each platform:
```

Comment:
```suggestion
This will navigate to the `index.html` file stored in the `WebContent` folder. This folder must be included in a platform-specific location on each platform:
```